helper_test.go

```diff
@@ -62,6 +62,34 @@ func createAgentPool(t *testing.T, client *Client, org *Organization) (*AgentPoo
 	}
 }
 
+func createAgentToken(t *testing.T, client *Client, ap *AgentPool) (*AgentToken, func()) {
+	var apCleanup func()
+
+	if ap == nil {
+		ap, apCleanup = createAgentPool(t, client, nil)
+	}
+
+	ctx := context.Background()
+	at, err := client.AgentTokens.Generate(ctx, ap.ID, AgentTokenGenerateOptions{
+		Description: String(randomString(t)),
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	return at, func() {
+		if err := client.AgentTokens.Delete(ctx, ap.ID); err != nil {
```

Comment:
Tests fail because this constructs a URL attempting to delete an agent token via a pool ID (e.g. `DELETE /api/v2/authentication-tokens/apool-XcSs6woTqgac3StZ`)

Reply:
🤦  It's not ap.ID, should be at.ID (Token ID). 👍